test/helpers.lua

```diff
@@ -202,10 +202,18 @@ function module.check_logs()
         fd:close()
         os.remove(file)
         if #lines > 0 then
-          local out = io.stdout
+          local out
+          if os.getenv('SYMBOLIZER') ~= nil then
+            out = io.popen(os.getenv('SYMBOLIZER'), 'w')
+          else
+            out = io.stdout
```

Comment:
nit: ternary avoids nesting, is more readable

Reply:
> nit: ternary avoids nesting, keeps the assignment as a single statement (helps readability)

Will clean this up.  Need to make it actually work, too. :)

Reply:
I have bad experiences with this sort of logic shortening in lua (is there a name for it?). I'd at least add braces:

```lua
local out = (os.getenv('SYMBOLIZER') == nil and io.stdout) or io.popen(os.getenv('SYMBOLIZER'), 'w')
```

Reply:
This wasn't quite working how I expected.  Took a different approach which actually [works](https://travis-ci.com/github/jamessan/neovim/jobs/381623621#L3901-L3936) but might need some prettying since Lua isn't my forté.